src/main/java/org/myrobotlab/io/FileIO.java

```diff
@@ -1424,5 +1426,35 @@ public static String normalize(String dirPath) {
       return dirPath.replace("\\", "/");
     }
   }
+  
+  public static boolean isExecutableAvailable(String command) {

```

Comment:
the method name here is misleading, some javadoc to describe the behavior please.  (potentially rename the method to   "attemptToRunCommand"    .. because it doesn't just check to see if the file is executable..

Reply:
:+1:

Reply:
updated, with javadoc, timeout, and default